crates/matrix-sdk/tests/integration/room/tags.rs

```diff
@@ -169,22 +169,20 @@ async fn when_set_is_low_priority_is_run_with_false_then_delete_tag_api_is_calle
 }
 
 #[async_test]
-async fn when_favorite_tag_is_set_in_sync_response_then_notable_tags_is_favorite_is_true() {
+async fn when_favorite_tag_is_set_in_sync_response_then_notable_tags_is_favourite_is_true() {
```

Comment:
Remove reference to notable tags in the test as its now private?
Also looks like the test is failing